magda-registry-api/src/main/scala/au/csiro/data61/magda/registry/RecordsService.scala

```diff
@@ -100,6 +141,20 @@ class RecordsService(config: Config, webHookActor: ActorRef, authClient: AuthApi
     }
   }
 
+  /**
+    * @apiGroup Registry Record Service
+    * @api {post} http://registry-api/records Create a new record
+    *
+    * @apiDescription Create a new record
+    *
+    * @apiParam (body) {json} record The definition of the new record.
+    *
+    * @apiHeader {string} X-Magda-Session Magda internal session id
+    *
+    * @apiSuccess (Success 200) {json} Response the record created
+    * @apiError (Error 400) {json} could not create
```

Comment:
This turns into 

![image](https://user-images.githubusercontent.com/900555/44707771-cc359b80-aae8-11e8-80fe-61c3d6ecff93.png)

Reply:
Updated~